internal/updater/updater.go

```diff
@@ -205,7 +197,11 @@ func (u *AvailableUpdate) InstallWithProgress(installTargetPath string, progress
 	return proc, nil
 }
 
-func (u *AvailableUpdate) event(label, version, msg string) {
+func (u *AvailableUpdate) analyticsEvent(action, label, version, msg string) {
+	if u.an == nil {
+		return
+	}
```

Comment:
I don't think this should ever be true? In which case if we were to encounter this we'd want it to error out, even panic.

Granted we should do nil checks but I feel like doing it at this level is excessive.